### PR TITLE
Add manual scaling to SDK

### PR DESCRIPTION
**Pull request by L-tech**

### CODE REVIEW

File: packages/compute/src/interfaces.ts [hunk 1]
```diff
@@ -152,6 +153,7 @@ interface InstanceCreationConfig {
     region: string;
     machineImageId?: string;
     storage: number;
+    akashMachineImageName?: string;
```
Comment (ilij4) [inline packages/compute/src/interfaces.ts:156]: we already have machineImageId property in this config. There is no need to add this as well

**Review by dusanstanis — DISMISSED** [review]

**Review by ilij4 — DISMISSED** [review]

**Review by rekpero — CHANGES_REQUESTED** [review]
make a check if the compute type is spot or on-demand, if spot throw error saying can't update these specs

**Review by rekpero — APPROVED** [review]

File: packages/compute/README.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ await client.instance.create(
       path: healthCheckPath,
       port: healthCheckPort,
     },
+    type: ComputeTypeEnum.DEMAND,
```
Comment (dusanstanis) [inline packages/compute/README.md:54]: Just check if u map this later to send it as :
scaling: true 
or 
scaling: false 

Cuz api supports only that. But it's fine to have it like this on your side, just make sure to map it properly @L-tech

Comment (L-tech) [reply packages/compute/README.md:54]: Yes. There is a mapping for it - https://github.com/spheronFdn/sdk/blob/96bcdf51d63e9c6dc1d38756970bd234fc712f4d/packages/compute/src/mappers.ts#L285 
@dusanstanis

**Pull request merged**